citi-onboarding/acervusej-onepage
# feat(feat/struture-style): Create the global style and structure

Things I've done:

-Download the font and its variations 
-Put the global colors that will be used on the website
-Place some standard styles, such as the font to be used, the class container with maximum size for each section, etc.
-Install the react-slick library

How to test it:

-First download the node modules on the client and the server. If you have already done that, then:
-Change the values on .env
-Go to client on the terminal (cd client) and put npm run start.
-On the browser write http://localhost:3000
-See if the font is equal and, if you want, add new things to the PostList.jsx to test all standards
-Look at the code to see if it has everything that should have
